libs/components/src/auth/AuthLogin.tsx

```diff
@@ -91,101 +98,120 @@ const AuthLogin = ({
-              formik.setFieldValue('password', e.target.value)
-            }
-            type={showPassword ? 'text' : 'password'}
-            size="medium"
-            endAdornment={
-              <InputAdornment position="end">
-                <IconButton
-                  aria-label="toggle password visibility"
-                  onClick={toggleShowPassword}
-                >
-                  {showPassword ? <VisibilityOff /> : <Visibility />}
-                </IconButton>
-              </InputAdornment>
-            }
-          />
-          {showSignIn && onForgotPassword ? (
-            <Link href="#" marginTop={1} onClick={onForgotPassword}>
-              Forgot password?
-            </Link>
-          ) : null}
-          <Button
-            type="submit"
-            disabled={loading}
-            variant="contained"
-            sx={{ marginTop: 3, width: '100%' }}
+
+      <form onSubmit={formik.handleSubmit}>
+        <TextInput
+          id="email"
+          placeholder="Email address"
+          size="medium"
+          value={formik.values.email}
+          hasError={!!formik.errors.email}
+          description={formik.touched.email ? formik.errors.email : undefined}
+          onBlur={formik.handleBlur}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            formik.setFieldValue('email', e.target.value)
+          }
+        />
+        <TextInput
+          id="password"
+          placeholder="Password"
+          value={formik.values.password}
+          hasError={!!formik.errors.password}
+          description={
+            formik.touched.password ? formik.errors.password : undefined
+          }
+          onBlur={formik.handleBlur}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            formik.setFieldValue('password', e.target.value)
+          }
+          type={showPassword ? 'text' : 'password'}
+          size="medium"
+          endAdornment={
+            <InputAdornment position="end">
+              <IconButton
+                aria-label="toggle password visibility"
+                onClick={toggleShowPassword}
+              >
+                {showPassword ? <VisibilityOff /> : <Visibility />}
+              </IconButton>
+            </InputAdornment>
+          }
+        />
+
+        {!showSignIn && formik.values.password.length ? (
+          <Box
+            sx={{
+              border: 1,
+              marginTop: 1,
+              padding: 1.5,
+              borderRadius: 1,
+              borderColor: 'grey.400',
+              '& .checklist-icon': {
+                marginTop: 0.5
+              },
+              fontSize: 14
+            }}
           >
-            Continue
-          </Button>
-        </form>
-      ) : null}
+            Your password must contain:
+            <PasswordChecklist
+              rules={['minLength', 'specialChar', 'number', 'capital']}
```

Comment:
this should be configurable, as chainlit app also use password auth and might not have those requiremetns

Reply:
Made configurable 👍